src/main/java/edu/ucsb/cs156/happiercows/entities/UserCommons.java

```diff
@@ -33,5 +33,7 @@ public class UserCommons {
   private int numOfCows;
 
   private double cowHealth;
+
+  private int cowsBought;
```

Comment:
It's not immediately obvious for new readers what the difference between `numOfCows` and `cowsBought` are as they can both mean either things. Something like `lifetimeNumCows` would be more readable.

Reply:
Chris has a good point.   I think perhaps you should change this.